_includes/modern-appendix.md

```diff
@@ -843,6 +851,32 @@ The text used in the last param of this message varies wildly.
 
 Indicates that there was a problem with a mode parameter. Replaces various implementation-specific mode-specific numerics.
 
+{% numericheader RPL_HELPSTART %}
+
+    "<client> <subject> :<firt line of help section>"
+
+Indicates the start of a reply to a {% command HELP %} command to the client.
+The text used in the last param of this message may vary, and SHOULD be displayed as-is by IRC clients to their users; possibly emphasized as a the title of the help section.
+
+The `<subject>` MUST be the one requested by the client, up to casing; unless it would be an invalid parameter, in which case it MUST be `*`.
+
+{% numericheader RPL_HELPTXT %}
+
+    "<client> <subject> :<line of help text>"
+
+When sending a reply to a {% command HELP %} command to the client, servers reply with each line of the help section as this numeric. Help lines MAY be wrapped to 80 characters by the server.
+
+The `<subject>` MUST be the one requested by the client, up to casing; unless it would be an invalid parameter, in which case it MUST be `*`.
+
+{% numericheader RPL_ENDOFHELP %}
+
+    "<client> <subject> :<last line of help text>"
+
+Indicates the end of the reply to the {% command HELP %} command to the client.
+The text used in the last param of this message may vary, and SHOULD be shown to the user as any other line.
```

Comment:
I don't think we need to explain this as much, that's covered by the examples in the linked `HELP` command. I think this matches the new text for `RPL_HELPTXT` pretty well and explains enough to get the point across.

```suggestion
Returns the final {% command HELP %} line to the client.
The text MUST be shown to the user as-is.
```

Reply:
same comment on the MUST

Reply:
removed the MUST!